docs/conf.py

```diff
@@ -212,14 +212,34 @@
         # A RTD thing to avoid a CSRF block when docs are using a custom domain
         hoverxref_api_host = "/_"
 
-hoverxref_tooltip_maxwidth = 600  # RTD main window is 696px
-hoverxref_auto_ref = True
-hoverxref_mathjax = True
-# hoverxref has to be applied to these
+hoverxref_auto_ref = False
 hoverxref_domains = ["py"]
-
+hoverxref_mathjax = True
+hoverxref_modal_hover_delay = 500
+hoverxref_tooltip_maxwidth = 600  # RTD main window is 696px
+hoverxref_intersphinx = [
```

Comment:
Is this just a list of all our intersphinx mappings or have you curated it?

Reply:
The first one.

Reply:
If we are going to do that then we can just do `list(intersphinx_mapping.keys())` or whatever, but should we curate it?

Reply:
Fair, I can do that instead. 

I personally think if we intersphinx, we would want them all to hover when we link to those classes.